src/query-builder/SelectQueryBuilder.ts

```diff
@@ -4236,9 +4237,9 @@ export class SelectQueryBuilder<Entity extends ObjectLiteral>
                     .join(" OR ")
             }
         } else {
-            const andConditions: string[] = []
+            let andConditions: string[] = []
```

Comment:
This change looks like a mistake. It can stay as `const`.

Reply:
Great catch, thanks!